### PR TITLE
Fix improper class name

### PR DESCRIPTION
**Pull request by joelip**
@bmneely just going to merge this since it's a fix for me being an idiot and not reading the entire `ClassFactory.build_response_object` method.

### CODE REVIEW

**Comment by joelip** [comment]
Actually...hmm. Does this seem like it should be an issue? I was getting this error when running the client:

```rb
=> #<NoMethodError: undefined method `new' for Checkpoints:Module>
[2] pry(#<CheckpointDeployer>)> e.backtrace
=> ["/Users/joelipper/.rvm/gems/ruby-2.1.7/gems/spartacus-0.2.8/lib/client/class_factory.rb:7:in `build_response_object'",
 "/Users/joelipper/.rvm/gems/ruby-2.1.7/gems/spartacus-0.2.8/lib/client/base_client.rb:19:in `block in convert_response'",
 "/Users/joelipper/.rvm/gems/ruby-2.1.7/gems/spartacus-0.2.8/lib/client/base_client.rb:19:in `map'",
 "/Users/joelipper/.rvm/gems/ruby-2.1.7/gems/spartacus-0.2.8/lib/client/base_client.rb:19:in `convert_response'",
 "/Users/joelipper/.rvm/gems/ruby-2.1.7/gems/spartacus-0.2.8/lib/client/checkpoints.rb:79:in `block in batch_create_checkpoints'",
 "/Users/joelipper/.rvm/gems/ruby-2.1.7/gems/spartacus-0.2.8/lib/client/base_client.rb:7:in `handle_timeouts'",
 "/Users/joelipper/.rvm/gems/ruby-2.1.7/gems/spartacus-0.2.8/lib/client/checkpoints.rb:70:in `batch_create_checkpoints'",
 "/Users/joelipper/Development/bloc_curriculum/official_roadmap_and_repos/curriculum/script/lib/checkpoint_deployer.rb:53:in `deploy!'",
 "script/deploy_new_checkpoints:40:in `deploy'",
 "script/deploy_new_checkpoints:52:in `<main>'"]
```

**Comment by getaaron** [comment]
What's the difference between `deploy` and `deploy!`?

**Comment by joelip** [comment]
Ok confirmed locally. Merging.

**Comment by bmneely** [comment]
One is said with twice the volume.

**Comment by joelip** [comment]
Dammit, I forgot to bump the gem version...

File: lib/client/checkpoints.rb [hunk 1]
```diff
@@ -76,7 +76,7 @@ def batch_create_checkpoints(roadmap_section_id=nil, checkpoints=[])
           checkpoints: checkpoints
         }
       )
-      convert_response(response, "checkpoints")
+      convert_response(response, "checkpoint")
```
Comment (mikesjewett) [inline lib/client/checkpoints.rb]: but what if we have more than one checkpoint? This is shortsighted.

Comment (bmneely) [inline lib/client/checkpoints.rb]: Bold stance from the boy who didn't remember to bump the gem version.

Comment (joelip) [inline lib/client/checkpoints.rb]: Thankfully, we live in a world where spite and memory are completely independent competencies

Comment (mikesjewett) [inline lib/client/checkpoints.rb]: yeah @joelip , remember there ain't nothing wrong... with a little bump (and grind).

**Pull request merged**